jpa/subsystem/src/main/java/org/jboss/as/jpa/container/NonTxEmCloser.java

```diff
@@ -22,12 +22,13 @@
 
 package org.jboss.as.jpa.container;
 
-import static org.jboss.as.jpa.messages.JpaLogger.ROOT_LOGGER;
+import org.wildfly.common.function.ThreadLocalStack;
 
+import javax.persistence.EntityManager;
 import java.util.HashMap;
 import java.util.Map;
 
-import javax.persistence.EntityManager;
+import static org.jboss.as.jpa.messages.JpaLogger.ROOT_LOGGER;
```

Comment:
@RanabirChakraborty The 'import static' should be first.

Our code style is:

import static

java.* (alphabetical)
javax.* (alphabetical)

other.* (alphabetical)

Reply:
Thanks! @bstansberry did the changes.